build.cake

```diff
@@ -691,6 +709,14 @@ Task("generate-mapping")
         $"  --output ./output/mappings/dependencies.json");
     using (var jsonReader = System.IO.File.OpenText("./output/mappings/dependencies.json")) {
         var o = (JObject)JToken.ReadFrom(new JsonTextReader(jsonReader));
+        // special case for the fact that we can't actually read the real nuget package yet
+        foreach (var p in o["packages"]) {
+            if (p["id"].Value<string>() == "Xamarin.AndroidX.Migration") {
+                p["dependencies"] = new JArray(new [] {
+                    "Xamarin.AndroidX.MultiDex"
+                });
+            }
+        }
```

Comment:
Because the actual build of the main solution only builds dummy nugets for the migration, we have to tweak the dependency tree. Once this is done, then we can actually build the migration nugets.